src/Uno.UI/UI/Xaml/Controls/Primitives/ToggleButton.cs

```diff
@@ -87,31 +87,6 @@ public bool IsThreeState
 		/// </summary>
 		internal bool CanRevertState { get; set; } = true;
 
-		protected virtual void OnIsCheckedChanged(bool? oldValue, bool? newValue)
-		{
-			// This workaround can be removed if pooling is removed. See https://github.com/unoplatform/uno/issues/12189
```

Comment:
Why is it removed now? Pooling is not removed yet

Reply:
@jeromelaban This whole method is unused. It's duplicated here:

https://github.com/unoplatform/uno/blob/d4929f000632c69a922344e65ea62ca24d58b3b6/src/Uno.UI/UI/Xaml/Controls/Primitives/ToggleButton.mux.cs#L228-L231

and the parameterless overload is the one that is actually used.